apps/omg/test/fixtures.exs

```diff
@@ -14,12 +14,12 @@
 
 defmodule OMG.Fixtures do
   use ExUnitFixtures.FixtureModule
+  import OMG.TestHelper
 
+  alias OMG.Eth.Configuration
   alias OMG.State.Core
 
-  import OMG.TestHelper
-
-  @eth OMG.Eth.RootChain.eth_pseudo_address()
+  @eth OMG.Eth.zero_address()
```

Comment:
`zero_address` was defined twice, once in RootChain module and the second time in OMG.Eth